.idea/vcs.xml

```diff
@@ -11,6 +11,6 @@
     </profile>
   </component>
   <component name="VcsDirectoryMappings">
-    <mapping directory="" vcs="Git" />
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
   </component>
 </project>
```

Comment:
It is generally recommended to end the files with newlines

Reply:
It's a IDE-config file